frontend/src/components/Metrics.tsx

```diff
@@ -33,13 +32,19 @@ export default function Metrics({
       if (!commitSHA) {
         if (!branch) return;
         commitSHA = await getCommitSHA(owner, repo, branch);
+        console.log(commitSHA);
       }
       try {
-        const [metricsBlob] = await getMetricsBlob(owner, repo, commitSHA);
-        const parsedData = parseMetrics(metricsBlob.content);
+        const [metricsBlob] = await getMetricsBlob(owner, repo, commitSHA, ["metrics.json"]);
+        //const parsedData = parseMetrics(metricsBlob.content);
+
+        // Converting the Buffer into a string
+        //const jsonString = JSON.stringify(metrics)
+        const parsedData = metricsFromJSON(metricsBlob.content);
+
         console.log("parsedData:", parsedData);
         setData(parsedData);
-        setSize(metricsBlob.size);
+        //setSize(metricsBlob.size);
```

Comment:
This was some kind of paper-related information or why is it not needed anymore?

Reply:
No still usefull, put it back in